internal/command/untaint.go

```diff
@@ -163,14 +165,24 @@ func (c *UntaintCommand) Run(args []string) int {
 		c.showDiagnostics(diags)
 		return 1
 	}
+
+	// Get schemas, if possible, before writing state
+	var schemas *terraform.Schemas
+	if isCloudMode(b) {
+		schemas, diags = c.GetSchemas(state, nil)
+		if diags.HasErrors() {
+			c.Ui.Warn(fmt.Sprintf(failedToLoadSchemasMessage, err))
```

Comment:
We don't really typically use `c.Ui.Warn` for new code because it generates yellow text that can be hard to read in some terminals.

In another comment elsewhere I suggested renaming `c.GetSchemas` to `c.MaybeGetSchemas` to try to be explicit that there's "something odd" about this function -- that it can potentially fail but still return successfully in some sense. In conjunction with that, I wonder about making that function itself return this warning in the `tfdiags.Diagnostics` it returns, so that we could then render it with our usual diagnostic formatting codepath. (`c.showDiagnostics(diags)`, similar to some of the examples above)